src/components/tools/table-tool/table-tool.scss

```diff
@@ -4,10 +4,14 @@
 $table-padding: 10px;
 $row-height: 34px;
 $index-column-width: 34px;
+$controls-column-width: 36px;
 $platform-scrollbar-width: 16px;
 $header-color: $workspace-teal-light-5;
+$header-input-color: #eef9fb;
 $border-style: 1px solid $charcoal-light-1;
+$border-input-style: 1px solid #dadada;
 $border-radius: 3px;
+$controls-hover-background: #c0dfe7;
```

Comment:
At some point if these colors are going to be used in other places, we might want to move them into vars.scss.  I don't see any duplicates at present, however.  Probably fine as is.